### PR TITLE
[16.0][IMP] account_statement_base: Show action to create Statements

### PR DESCRIPTION
**Pull request by carlos-lopez-tecnativa**
Before this commit, when a user clicked the "New Transaction" button, an error was raised. After this commit, statement lines are displayed to create new records.

@Tecnativa TT50459

@pedrobaeza  @carolinafernandez-tecnativa could you please take a look

### CODE REVIEW

File: account_statement_base/models/account_journal_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    _inherit = "account.journal"
+
+    def create_cash_statement(self):
+        self.ensure_one()
+        action = self.env["ir.actions.actions"]._for_xml_id(
+            "account_statement_base.account_bank_statement_line_action"
+        )
+        action["context"] = {
+            "search_default_journal_id": self.id,
+        }
```
Comment (pedrobaeza) [inline account_statement_base/models/account_journal_dashboard.py]: Without the final comma, everything goes to one line:

```suggestion
        action["context"] = {"search_default_journal_id": self.id}
```

File: account_statement_base/models/account_journal_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from odoo import models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    def create_cash_statement(self):
+        self.ensure_one()
```
Comment (pedrobaeza) [inline account_statement_base/models/account_journal_dashboard.py:11]: Please always document the why of the things:

```suggestion
        # Totally override this action for avoiding the standard
        # message saying that you need to install the enterprise
        # module. We do the equivalent thing instead.
        self.ensure_one()
```

**Review by pedrobaeza — COMMENTED** [review]
There's no runboat to try though. Let's see in the next build.

File: account_statement_base/models/account_journal_dashboard.py (hunk 2 above)
Comment (carlos-lopez-tecnativa) [reply account_statement_base/models/account_journal_dashboard.py:11]: Thanks, I only added a similar comment in the commit message but not in the function itself. I will take that into account next time.

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM, Thanks

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**